common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -195,19 +195,21 @@ def test_all_sections_and_fields_are_present(self):
             },
             {
                 'title': 'Social Media Links',
-                'fields': [
+                'fields': sorted([
                     'Twitter Link',
                     'Facebook Link',
                     'LinkedIn Link',
```

Comment:
There's actually a bug in the application code where these should be shown on the page in a consistent order but won't be under Python 3.5, but Marco says that's ok for now.